test/judges/test-github-events.rb

```diff
@@ -917,12 +917,22 @@ def test_pull_request_event_with_comments
 
   def test_count_numbers_of_workflow_builds
     fb = Factbase.new
-    load_it('github-events', fb, Judges::Options.new({ 'repositories' => 'foo/foo', 'testing' => true }))
-    f = fb.query('(eq what "pull-was-merged")').each.to_a.first
+    repo = 'zerocracy/baza'
+    load_it('github-events', fb, Judges::Options.new({ 'repositories' => repo, 'testing' => true }))
+    f = fb.query("(and (eq what \"pull-was-merged\") (eq branch \"#{repo}\"))").each.to_a.first
```

Comment:
@Suban05 here the branch name is exactly the repository name?

Reply:
@Yegorov in the tests yes, but in the real data - no. maybe, filter by `event_id`?
like this:
`fb.query('(and (eq what "pull-was-merged") (eq event_id 42))')`

Reply:
@Suban05 perhaps yes